contributed_definitions/NXenergydispersion.nxdl.xml

```diff
@@ -69,19 +69,79 @@
     </field>
     <field name="energy_scan_mode" type="NX_CHAR">
         <doc>
-             Way of scanning the energy axis (fixed or sweep).
-             
-                 This concept is related to term `12.65`_ of the ISO 18115-1:2023 standard.
-             
-             .. _12.65: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.65
-             
-                 This concept is related to term `12.66`_ of the ISO 18115-1:2023 standard.
-             
-             .. _12.66: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.66
+             Way of scanning the energy axis
         </doc>
         <enumeration>
-            <item value="fixed"/>
-            <item value="sweep"/>
+            <item value="fixed_analyser_transmission">
+                <doc>
+                     constant ΔE mode, where the electron retardation (i.e., the fraction of pass energy to
+                     kinetic energy, R = (EK - WF/Ep)) is scanned, but the pass energy Ep is kept constant.
+                     Here, WF is the spectrometer work function.
+                     This mode is often used in XPS/UPS because the energy resolution does not change with
+                     changing energy (due to the constant pass energy).
+                     
+                     Synonyms: constant ΔE mode, constant analyser energy mode, CAE mode, FAT mode
+                     
+                         This concept is related to term `12.64`_ of the ISO 18115-1:2023 standard.
+                     .. _12.64: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.64
```

Comment:
Something, I recently encountered while fixing the parsing of iso paragraphs was that enumeration items are rendered into a single line. Previously, it did even not build the documentation when there was a documentation with more than one paragraph. Now it just concatenates everything.

So we need to find a solution for this rather long docstrings here as this will be horrible to read I think. But probably this is something we need to fix in the html generation.

Reply:
It's a bit cluttered but is fine for now I think.

<img width="1511" alt="Bildschirmfoto 2024-01-05 um 16 27 16" src="https://github.com/FAIRmat-NFDI/nexus_definitions/assets/1014422/dbe7af12-b2d2-4434-a9ea-6b05eae3ecb9">

Reply:
Ok, that is indeed a bit cluttered :D is that on nyaml or on the HTML generation? In non-xref cases, the HTML build does recognize paragraph breaks, right?

Reply:
See #142